pandas/tests/groupby/test_groupby.py

```diff
@@ -134,6 +134,23 @@ def func(dataf):
     result = df.groupby("X", squeeze=False).count()
     assert isinstance(result, DataFrame)
 
+    # https://github.com/pandas-dev/pandas/issues/28330
+    # Test groupby operations on subclassed dataframes/series
+
```

Comment:
actually these tests *are* different that the above ones, can you make a new test